### PR TITLE
chore(release): migrate deployment examples to 7.16

### PR DESCRIPTION
**Pull request by tasso94**
* spring-wildfly-non-pa
* spring-servlet-pa-wildfly
* spring-servlet-pa
* spring-boot
* servlet-pa
* embedded-spring-rest
* ejb-par

related to CAM-13909

### CODE REVIEW

File: deployment/embedded-spring-rest/pom.xml [hunk 1]
```diff
@@ -10,9 +10,10 @@
 
   <properties>
     <version.camunda>7.16.0</version.camunda>
-    <version.spring>5.2.8.RELEASE</version.spring>
+    <version.spring>5.2.14.RELEASE</version.spring>
```
Comment (tmetzke) [inline deployment/embedded-spring-rest/pom.xml:13]: Should we go for the latest Spring 5 (i.e., 5.3.10)?

File: deployment/embedded-spring-rest/pom.xml [hunk 2]
```diff
@@ -10,9 +10,10 @@
...
     <version.resteasy-jaxrs>3.14.0.Final</version.resteasy-jaxrs>
-    <version.h2>1.3.171</version.h2>
+    <version.h2>1.4.190</version.h2>
+    <version.slf4j>1.7.26</version.slf4j>
```
Comment (tmetzke) [inline deployment/embedded-spring-rest/pom.xml:16]: Should we go for the latest versions here?
H2: 1.4.200
SLF4J: 1.7.32

File: deployment/embedded-spring-rest/src/main/java/org/camunda/bpm/example/loanapproval/CalculateInterestService.java [hunk 3]
```diff
@@ -19,12 +19,16 @@
...
-		
-	}
+
+  protected static final Logger LOG = Logger.getLogger(CalculateInterestService.class.getName());
+
+  public void execute(DelegateExecution delegate) {
+
+    LOG.info("Spring Bean invoked.");
+
+  }
```
Comment (tmetzke) [inline deployment/embedded-spring-rest/src/main/java/org/camunda/bpm/example/loanapproval/CalculateInterestService.java:32]: Nice 👍

File: deployment/spring-servlet-pa-tomcat/pom.xml [hunk 4]
```diff
@@ -10,7 +10,13 @@
   <properties>
     <camunda.version>7.16.0</camunda.version>
     <spring.version>5.3.10</spring.version>
-    <tomcat.version>9.0.43</tomcat.version>
+    <tomcat.version>9.0.52</tomcat.version>
+    <servlet-api.version>3.0.1</servlet-api.version>
+    <h2.version>1.4.190</h2.version>
+    <slf4j.version>1.7.26</slf4j.version>
```
Comment (tmetzke) [inline deployment/spring-servlet-pa-tomcat/pom.xml:16]: See latest versions above for H2 and SLF4J

File: deployment/embedded-spring-rest/pom.xml (hunk 1 above)
Comment (tasso94) [reply deployment/embedded-spring-rest/pom.xml]: 👍

File: deployment/embedded-spring-rest/pom.xml (hunk 2 above)
Comment (tasso94) [reply deployment/embedded-spring-rest/pom.xml]: 👍

File: deployment/spring-servlet-pa-tomcat/pom.xml (hunk 4 above)
Comment (tasso94) [reply deployment/spring-servlet-pa-tomcat/pom.xml]: 👍

**Review by tmetzke — APPROVED** [review]

**Pull request merged**